metaflow/plugins/kfp/kfp.py

```diff
@@ -878,7 +900,7 @@ def kfp_pipeline_from_flow(
 
             def build_kfp_dag(
                 node: DAGNode,
-                passed_in_split_indexes: str = '""',
+                passed_in_split_indexes: str = "",
```

Comment:
why this change?

Reply:
The default value here is really saying no split index, which is much more intuitively an empty string. The quotation literals are only for bash code formatting and this formatting should be living in bash code. 

`passed_in_split_indexes` is also being used in mflog config code where I do want an empty string instead of a pair of quotation marks.